client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -198,9 +196,7 @@ class ThemeEnhancements extends Component {
 					status="is-info"
 					showDismiss={ false }
 					text={
-						minilevenModuleActive &&
-						jetpackVersion &&
-						versionCompare( jetpackVersion, '8.1-alpha', '>=' )
+						minilevenModuleActive
```

Comment:
Can we maybe remove the entire `renderMinilevenSettings` UI? Since 8.3-alpha, the `siteHasMinileven` value will always be `false` and the `minileven` settings won't be shown. Or do we still want to show it to `>=8.0 && < 8.3` users.

Reply:
Good point 👍  

I was planning to remove that separately, but you're right. This has been deprecated for a while now (over a year), so removing as well. 

Done in 0c92425c936f8897c6a29e17fb6e5c274d9b8027